### DISCUSSION TITLE
Error related to aerostrcutural optimization

### DISCUSSION THREAD

**Discussion by Aming12-01** [mdolab/dafoam#700] [General]
hello, I am trying to run an aerostructural optimization of an high aspect ratio wing, however, i ran into an error says that the Design variables have no impact on the constraints or objective. can i please know what might the reason of this error and some possible solution  to solve it 
thank you 
 
![Weixin Image_20241031163644](https://github.com/user-attachments/assets/0132c4ec-f3d1-46fd-aed0-186975f3a377)

**Comment by friedenhe** [comment]
Do you have this problem when running the aero-structural tutorial from DAFoam (MACH-Tutorial wing)? We would need more detailed info to look into the problem.
